src/node-lib/curriculum-api/index.ts

```diff
@@ -399,13 +399,24 @@ const curriculumApi = {
       programmesUnavailable,
     } = transformMVCase(res);
 
+    let unavailableProgrammesWithoutAvailable;
```

Comment:
Is this the same code that you worked on in your other PR?

Reply:
Pushed to the wrong branch, thats gone now